<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UICmdDebug</class>
 <widget class="QWidget" name="UICmdDebug">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit_DEBUG">
     <property name="centerOnScroll">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkBoxCL_IN">
       <property name="text">
        <string>CL_IN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxCL_OUT">
       <property name="text">
        <string>CL_OUT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxHUB_IN">
       <property name="text">
        <string>HUB_IN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxHUB_OUT">
       <property name="text">
        <string>HUB_OUT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxDHT_IN">
       <property name="text">
        <string>DHT_IN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxDHT_OUT">
       <property name="text">
        <string>DHT_OUT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxFilterIP">
       <property name="text">
        <string>Filter by IP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditIP"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Lines:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxLines">
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
